sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_backup_client.py

```diff
@@ -37,15 +47,44 @@ def begin_backup(self, blob_storage_url, sas_token, **kwargs):
         :returns: An :class:`~azure.core.polling.LROPoller` instance. Call `result()` on this object to wait for the
             operation to complete and get a :class:`KeyVaultBackupOperation`.
         :rtype: ~azure.core.polling.LROPoller[~azure.keyvault.administration.KeyVaultBackupOperation]
+
+        Example:
+            .. literalinclude:: ../tests/test_examples_administration.py
+                :start-after: [START begin_backup]
+                :end-before: [END begin_backup]
+                :language: python
+                :caption: Start a vault backup operation
+                :dedent: 8
         """
         polling_interval = kwargs.pop("_polling_interval", 5)
         sas_parameter = self._models.SASTokenParameter(storage_resource_uri=blob_storage_url, token=sas_token)
+
+        continuation_token = kwargs.pop("continuation_token", None)
+        status_response = None
+        if continuation_token:
+            try:
+                job_id = _parse_status_url(base64.b64decode(continuation_token.encode()).decode("ascii"))
+            except:  # pylint: disable=broad-except
+                raise ValueError("The provided continuation_token is malformed. A base64-encoded URL is expected")
```

Comment:
I think it would be better to reference the poller's `continuation_token()`, if anything (simply stating the token is malformed works for me), to avoid implying a user could, or perhaps in some cases should, construct a token.

Reply:
That's fair -- makes sense to keep the token fully opaque. What do you mean by referencing the poller's `continuation_token()`?

Reply:
I mean that if we suggest a user action or root cause here, it should be to the effect that `poller.continuation_token()` is how one gets a valid token.